test/e2e/apps/replica_set.go

```diff
@@ -524,3 +534,138 @@ func testRSLifeCycle(f *framework.Framework) {
 
 	framework.ExpectNoError(err, "failed to see replicas of %v in namespace %v scale to requested amount of %v", rs.Name, ns, rsPatchReplicas)
 }
+
+func testRSStatus(f *framework.Framework) {
+	ns := f.Namespace.Name
+	c := f.ClientSet
+	rsClient := c.AppsV1().ReplicaSets(ns)
+
+	// Define ReplicaSet Labels
+	rsPodLabels := map[string]string{
+		"name": "sample-pod",
+		"pod":  WebserverImageName,
+	}
+	labelSelector := labels.SelectorFromSet(rsPodLabels).String()
+	framework.Logf("labels: %v", labelSelector)
```

Comment:
is this needed? if so, could there be a longer description message?

Reply:
Removed.